content/swagger/resources/products/definitions/product.yaml

```diff
@@ -151,10 +151,10 @@ properties:
     x-from-version: '5.0'
   completenesses:
     type: array
-    description: Product completenesses for each channel/locale combination (only available on SaaS platforms, and when the "with_completenesses" query parameter is set to "true")
+    description: Product completenesses for each channel/locale combination (only available since 7.0, and when the "with_completenesses" query parameter is set to "true")
```

Comment:
```suggestion
    description: Product completenesses for each channel/locale combination (only available since the 7.0 version, and when the "with_completenesses" query parameter is set to "true")
```